testing/src/cypress/custom-command/common/authorization.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { AxiosResponse } from "axios";
+import { stringify } from "querystring";
+import authData from "../../e2e/data/authentication-details.json";
+
+export default class Authorization {
+  async getAuthToken(): Promise<AxiosResponse<string>> {
+    const auth_url = authData.testEnv.keyCloakTokenUrl;
+    const body = stringify({
+      username: authData.testEnv.username,
```

Comment:
I believe that `username`, `password`, and `client_id` would likely be received as parameters to be read from an env config or JSON file, right?

Reply:
moved the auth details to cypress env file.